src/main/java/org/inferred/freebuilder/processor/util/Excerpts.java

```diff
@@ -103,6 +105,35 @@ public static Excerpt forEach(TypeMirror elementType, String iterable, String me
     return new ForEachExcerpt(elementType, iterable, method);
   }
 
+  private static final class GeneratedAnnotationExcerpt extends Excerpt {
+    private final Class<?> generator;
+
+    GeneratedAnnotationExcerpt(Class<?> generator) {
```

Comment:
you should probably have a test that the Generated annotation doesn't show up in JDK9. Or, is that way more work?